section2/exercises/if-statements.js

```diff
@@ -62,11 +70,14 @@ When numQuarters = 2, program should log "I have enough money for a gumball"
 When numQuarters = 3, program should log "I have enough money for a gumball"
 */
 
-var numQuarters = 0;
+var numQuarters = 4;
 
 // Write your conditional here
-console.log("I have enough money for a gumball");
-console.log("I don't have enough money for a gumball");
+if (numQuarters >= 2){
+  console.log("I have enough money for a gumball");
+} else if (numQuarters < 2){
```

Comment:
This _could_ be just an `else` without the condition; since anything less than 2 wouldn't meet the criteria of the initial `if`.